oidc_provider/lib/endpoints/authorize.py

```diff
@@ -112,13 +117,15 @@ def validate_params(self):
             raise AuthorizeError(self.params['redirect_uri'], 'invalid_request', self.grant_type)
 
         # Response type parameter validation.
-        if self.is_authentication and self.params['response_type'] != self.client.response_type:
+        if self.is_authentication \
```

Comment:
I'd like to see this match the style of the other `if`s in the file and not use a line continuation.

Reply:
Discussed elsewhere. Going to hold off on formatting changes as this is a fork and it may make it harder to maintain from upstream later. Unless anyone else feels strongly enough we should make any of these changes.